pkg/schedule/schedulers/balance_key_range.go

```diff
@@ -0,0 +1,151 @@
+package schedulers
```

Comment:
Need a license and please also change the filename.

Reply:
done